### ISSUE TITLE
Native Tracker - TikTok (Fingerprinting)

### ISSUE THREAD

**Issue by nightznero** [hagezi/dns-blocklists#576]
Noticed that u Hagezi have missing few lines that can be easily blocked
Best regards!
#### Which domain(s) should be blocked?

0.0.0.0 sf16-muse-va.ibytedtos.com
0.0.0.0 ads-sg.tiktok.com
0.0.0.0 ads.tiktok.com

**Comment by hagezi** [comment]
Thanks, as far as I know Tiktok links and videos are not loaded correctly from the browser if you block `sf*-muse-va.ibytedtos.com`.
Do you have more details why this should be blocked, or what exactly is being transmitted?

I have included the others.

**Comment by hagezi** [comment]
I searched 30 days logs of the whole family, some TikTok users are there. I did not find any call from `sf*-muse-va.ibytedtos.com`. I also don't know how to trigger it to investigate what is being transferred.

**Comment by nightznero** [comment]
> I searched 30 days logs of the whole family, some TikTok users are there. I did not find any call from `sf*-muse-va.ibytedtos.com`. I also don't know how to trigger it to investigate what is being transferred.

I was look at TikTok and thats what i find. (It was litellary 5min test but think its enought) `sf16-muse-va.ibytedtos.com` is 3rd url after open tiktok app
![Screenshot_20230302_063300_AdGuard](https://user-images.githubusercontent.com/61750641/222342318-bf1acb6b-f1cc-45d3-80e5-316128a111f0.jpg)
(Didnt whitelisted installation id yet so ignore it) While scrolling,commenting,liking,opening liked/saved vids and so on,didn't noticed again "sf" link. But after closing and open again it was again 3rd link. I think is just an Analyzer to which first vid should be recommended to you. It's just my theory but see lot sense there cuz if later didn't noticed that url it's just put you in some wall/feed/bubble and that's all. That's all I noticed and can provide,if feel not to add that's fine! Still 2urls has been added so I'm happy with dat😅
